hawc/apps/lit/managers.py

```diff
@@ -351,6 +352,18 @@ def prune_tags(self, root_tag, pruned_tags, descendants: bool = False):
         )
         return self.exclude(query).distinct("pk")
 
+    def user_tags(self, user_id: int) -> dict[int, list[int]]:
```

Comment:
I changed how we're building this dictionary for the serializer since it looked like the old way required a query for each reference if user references were found